project/src/com/google/daggerquery/dagger_query_textproto.bzl

```diff
@@ -25,7 +25,7 @@ def _dagger_query_textproto_impl(ctx):
     ctx.actions.run_shell(
         inputs = src_jars,
         outputs = [ctx.outputs.out],
-        command = "unzip {src_jar} '*_graph.textproto' -d .; zip {out} '*_graph.textproto'".format(
+        command = "unzip {src_jar} '*_graph.textproto' -d graphs; cd graphs; zip -r ../{out} .".format(
```

Comment:
Nit: would you mind making this a bit more descriptive?
eg. binding_graphs_tmp

Reply:
Thanks, good idea ✨✔️